packages/compose/README.md

```diff
@@ -291,10 +291,14 @@ throws a warning when using useLayoutEffect in that environment.
 
 Attach a keyboard shortcut handler.
 
+_Related_
+
+-   <https://craig.is/killing/mice#api.bind> for information about the `callback` parameter.
+
 _Parameters_
 
 -   _shortcuts_ `string[]|string`: Keyboard Shortcuts.
--   _callback_ `Function`: Shortcut callback.
+-   _callback_ `import('mousetrap').MousetrapInstance['bind']`: Shortcut callback.
```

Comment:
Isn't this just a DOM event callback?

Reply:
It'sa not quite, from the types:

```js
        bind(
            keys: string | string[],
            callback: (e: ExtendedKeyboardEvent, combo: string) => any,
            action?: string,
        ): this;
```

Reply:
Shouldn't the callback be of type `(e: ExtendedKeyboardEvent, combo: string) => any` ?

Reply:
Oh shoot you're right, the callback is passed as the second param to `bind`, my bad. I'll fix this.